stroke.py

```diff
@@ -0,0 +1,27 @@
+import requests
+import re
+import os
+import time
+
+def get_stroke(hanzi):
+  return '&nbsp;'.join(map(get_stroke_single, hanzi))
+
+def get_stroke_single(hanzi):
+  if not os.path.exists('content/'):
+    os.mkdir('content/')
+
+  hanzi_id = str(ord(hanzi))
+  gif_loc = 'content/' + hanzi_id + '.gif'
```

Comment:
why not use the hanzi in the filename? modern systems can support UTF8

Reply:
and this is python3, so should be fine!

Reply:
also could be f string

Reply:
Seems safer to use ID instead of Unicode, dunno. Seeing unicode in filenames makes me nervous... I suppose it makes more sense for searchability reasons, though. I can try and see if it makes Anki blow up.

Reply:
Oh, I originally tried running this project in Py3 and it blew up so I had assumed it was Py2. Will investigate.

Reply:
https://github.com/samm81/chinese-anki-helper-script/blob/master/Pipfile#L20